### ISSUE TITLE
S3 repo url returns 404 causing 01_update_platforms_check.sh failing

### ISSUE THREAD

**Issue by de1987** [fedora-iot/greenboot#71]
Our edge devices are getting this issue below when logging into the device using SSH.
```
Script '01_update_platforms_check.sh' FAILURE (exit code '1'). Continuing...
Boot Status is GREEN - Health Check SUCCESS
```
It sounds like, the same issue occurred on this opened issue https://github.com/fedora-iot/greenboot/issues/68
Since ostree repos are common for every system which uses it.
Our idea was to change to script to check for the `config` file instead of the root directory.

Is there another way to fix it for different repos flavors we have?

**Comment by nullr0ute** [comment]
The idea is that it checks to see if can retrieve an update. What is the config you're referring to?

**Comment by de1987** [comment]
@nullr0ute yeah, but it's checking the root directly of the repo, which in most of the cases has access blocked.
We noticed that ostree repos have a `config` file on it which it's acessible.

On issue #68 if you tried to open this URL https://ostree.fedoraproject.org/iot/config, you are able to check that the repo is available to retrieve an update.
And for edge management, it's the same, but there we have the repos on AWS S3.

**Comment by britthouser** [comment]
I am seeing similar issue:
```
[tecumseh@tecumseh ~]$ sudo /usr/lib/greenboot/check/wanted.d/01_update_platforms_check.sh
There are problems connecting with the following URLs:
https://ostree.fedoraproject.org/iot/
[tecumseh@tecumseh ~]$ curl https://ostree.fedoraproject.org/iot/
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>403 Forbidden</title>
</head><body>
<h1>Forbidden</h1>
<p>You don't have permission to access this resource.</p>
<hr>
<address>Apache Server at ostree.fedoraproject.org Port 443</address>
</body></html>
```
It seems like the apache config has change?
